src/generator/AutoRest.Java.Azure.Fluent/Model/MethodGroupJvaf.cs

```diff
@@ -28,7 +28,19 @@ public MethodGroupJvaf(string name) : base(name)
         public override string ImplClassSuffix => "Inner";
 
         [JsonIgnore]
-        public override string ParentDeclaration => "";
+        public override string ParentDeclaration
```

Comment:
Fixes - Azure/azure-sdk-for-java#1488

Search for public class ContainerServicesInner implements InnerSupportsListing<ContainerServiceInner> {